index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   akka-operator:
   - apiVersion: v2
```

Comment:
what is apiVersion? is that the api of Helm or our operator?

Reply:
ours

Reply:
what do we have that is v2? the crd is v1 and the image/chart version is 0.4.2

Reply:
I think it is chart api version

Reply:
also index.yaml is generated

Reply:
@andreaTP ? in cloudflow index this is all set to v1..

Reply:
Is generated, I'll check after lunch

Reply:
Comes from `Chart.yaml`:
https://github.com/lightbend/akka-operator-helm/blob/master/akka-operator/Chart.yaml#L1

Should we revert it to v1 and re-publish?

Reply:
No, it indicates that we mandate Helm3 and I guess is fair to do so:
https://helm.sh/docs/topics/charts/#the-apiversion-field

Reply:
Makes sense to correlate that to the v1 in the crd.

We can do that and release 0.4.3 (and remove 0.4.2).

Reply:
final call?
add a comment there for future generations.

Helm 3 is what we want to support.

Reply:
done